semana7/projeto-spotif4/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Evite estilizar em um arquivo geral usando o React. Isso pode causar alguns problemas se não for muito bem feito e com cuidado! Tente usar mais o styled-components!

Pelo que entendi, a maior parte desse código já vinha com o React né? Lembre-se de apagar antes de subir! É ruim deixar um código de CSS que não fomos nós quem fizemos!